.chloggen/rename_remoteobserverprocessor.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
is it a breaking change if the module was never tagged?

Reply:
It was tagged a couple times